src/App.js

```diff
@@ -7,16 +7,20 @@ import RoverPage from './features/RoverPage/RoverPage';
 import { useDispatch } from 'react-redux';
 import { setRovers } from './app/rootSlice';
 import './App.css';
-import fetchRovers from './features/Apicalls';
+import {fetchRovers} from './features/Apicalls';
 
 function App() {
   const dispatch = useDispatch()
-  
-  useEffect(async () => {
+  const fetchData = async () => {
     const roverData = await fetchRovers()
     dispatch(setRovers(roverData.rovers))
-  }, [])
+  }
```

Comment:
Good use of async/await.   Be sure that everyone one in the group fully understands async/await - to the point that they could explain it to me if asked.   *Each group member should take ownership of doing some solo study themself to ensure they understand this - the documentation, a blog post, a youtube video, talking to their house mentor about it, etc.    But the group as a whole should continue to check in on those efforts and help support each other in their understanding as well.